ceph-releases/ALL/debian/daemon-base/__DOCKERFILE_INSTALL__

```diff
@@ -0,0 +1,8 @@
+# add the necessary repos
+    echo "deb http://obs.linaro.org/ERP:/ceph-ansible:/luminous/Debian_9 ./" \
```

Comment:
This is where you should use ${CEPH_VERSION} to replace luminous and __ENV_[BASEOS_TAG]__ instead of 9.